src/js/components/mixins/SignalValue.jsx

```diff
@@ -74,7 +74,9 @@ module.exports = {
     }
 
     if (signal) {
-      sg.set(signal, value);
+      // Set the signal on the view but do not dispatch: the `.signal` listener
+      // above will dispatch the action to synchronize Redux with Vega.
+      sg.set(signal, value, false);
```

Comment:
I went back and forth on whether to make this behavior default but I think that having this be "false" (and implicitly "true" if not set) leads to the clearest mental picture of the flow of operations within this file